src/components/Auth/TodoInfo.tsx

```diff
@@ -0,0 +1,124 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Loader } from './Loader';
+
+type Props = {
+  todo: Todo;
+  updateTODOCompleted: (todoId: number | undefined, completed: boolean) => void;
```

Comment:
the parent component expected other arguments for this function
```suggestion
  updateTODOCompleted: (todoNew: Todo) => void;
```